examples/webhook-notification/index.js

```diff
@@ -0,0 +1,47 @@
+require('dotenv').config();
+const { Defender } = require('@openzeppelin/defender-sdk');
+const ReqBody = require('./body.json');
+
+const WEBHOOK_SECRET = process.env.WEBHOOK_SECRET || 'c7ad9ef4276dac354cb51d3aa9b1e4c7';
+
+function webhook(req, res) {
+  const creds = {
+    apiKey: process.env.API_KEY,
+    apiSecret: process.env.API_SECRET,
+  };
+  const client = new Defender(creds);
+
+  const signature = req.headers['Defender-Signature'];
+  const timestamp = req.headers['Defender-Timestamp'];
+
+  const result = client.notificationChannel.verifySignature({
+    body: req.body,
```

Comment:
what about just passing whole request and underlying logic is responsible for extracting what is needed?

This way users do not need to know exact names of headers.

Reply:
We just need the body from the request, the rest of the values are not present in the req, also that would make our API very attached to the framework, I'm not aware of other node Rest libraries, but what if the library gets the body using a function like:
```js
const body = req.getBody()
```
instead of `req.body`

Reply:
Same with headers